### ISSUE TITLE
Spatially varying gravity does not appear to work

### ISSUE THREAD

**Issue by dvswenson** [firemodels/fds#3001]
I have run a model that essentially replicates the example given in Section 6.4.6 Special Topics: Defying Gravity. I tested with FDS 6.2 and 6.3. I did modify the vector to put it at a 45 degree angle.

 Version          : FDS 6.2.0
 SVN Revision No. : 22343

My model has particles and the streakline plots show that the particles all moved using a constant gravity vector. The particles did not show any spatial change of gravity.

Sincerely,
Daniel Swenson
Thunderhead Engineering
![test_gravity_manual fds](https://cloud.githubusercontent.com/assets/13338306/10764514/a9e9b21e-7c9c-11e5-8ad6-127c285d1eed.png)
![gravity streaklines manual](https://cloud.githubusercontent.com/assets/13338306/10764531/b8b890c6-7c9c-11e5-9ce0-7fc0a1742be1.png)

**Comment by rmcdermo** [comment]
Thanks, Dan.  I'll have a look.

**Comment by rmcdermo** [comment]
Definitely, the gravity ramp is not hooked up to particles.  I am fixing this asap.

**Comment by dvswenson** [comment]
Another comment is that Smokeview appears to use the gravity vector as a reference when displaying the model. It becomes very hard to rotate the model in this coordinate system.

**Comment by rmcdermo** [comment]
This should be fixed with pull request #3011.  Here is a test case.

[test_gravity_manual.fds.txt](https://github.com/firemodels/fds-smv/files/22765/test_gravity_manual.fds.txt)

Dan, note that it is more convenient (at present) to specify GVEC=-1,0,-1 and then to use + in the ramp.  Otherwise, Smokeview draws things upside down.  I don't think we need to make SMV check the local gravity vector in the FDS loops.  That would be a mess.

Glenn is having a look at the rotation issue.

**Comment by rmcdermo** [comment]
This particle issue is pretty significant, in my opinion.  We should probably start thinking about a maintenance release in a few weeks or so.

We can start scheduling our releases based on whenever Dan submits an Issue!

**Comment by dvswenson** [comment]
Thank you for fixing this.

**Comment by gforney** [comment]
open up the Specify Orientation tab in the Motion dialog box.  Then use the widgets there to control the scene.

I'll take a look at getting scene motion to behave better with just the mouse

**Comment by gforney** [comment]
I had a couple of rotations reversed.  The windows smokeview at the following link should work better.  
https://drive.google.com/folderview?id=0B_wB1pJL2bFQc1F4cjJWY2duWTA&usp=sharing

**Comment by gforney** [comment]
have you had a chance to try the new smokeview?

**Comment by dvswenson** [comment]
I just did, but am a bit confused since it appears very similar to what it was previously. When I start Smokeview with the gravity model listed above, it looks like the attached image. 

My vote would be for Smokeview to use the model coordinates for the geometry and not use coordinates that are associated with gravity.

I did use Glenn's link to download the new version, but maybe that was not updated?

Dan

![smokeview_initial_display](https://cloud.githubusercontent.com/assets/13338306/10970883/eaa8cc58-8396-11e5-8967-0570df944051.png)
